### DISCUSSION TITLE
Regarding Installation

### DISCUSSION THREAD

**Discussion by prasant5789** [GRIDAPPSD/GOSS-GridAPPS-D#1747] [Installation] [answered]
**Problem while installing GridAPPS-D**

I am trying to install GridAPPS-D.  I followed the following steps mentioned in its website:

- git clone https://github.com/GRIDAPPSD/gridappsd-docker

- cd gridappsd-docker
- ./docker_install_ubuntu.sh

Now when I tried to run command ./run.sh. It shows the following output.

_Note: I have installed ubuntu on WSL2 on Windows 11._ 

<img width="879" alt="Screenshot 2023-08-04 171849" src="https://github.com/GRIDAPPSD/GOSS-GridAPPS-D/assets/112493485/a96acc0d-6ca2-424b-947c-b2b3307a4eb6">

**Comment by poorva1209** [comment, marked as answer]
Do you have Docker Desktop installed on your Windows system? If not, here is the link for installing: https://gridappsd-training.readthedocs.io/en/latest/installation/windows10/1.1W-Virtual-Machine-Docker-Setup.html#Install-Docker-for-Windows

**Reply by prasant5789** [reply]
Thank you! It works now.
